osafw-app/App_Data/template/common/form/showform/select.html

```diff
@@ -1,4 +1,4 @@
-<select id="<~field>" name="item[<~field>]" class="form-select <~class_control>" <~attrs_control noescape>>
+<select id="<~field>" name="item[<~field>]" class="form-select <~class_control>" <~/common/required if="required"> <~attrs_control noescape>>
```

Comment:
are you sure required is here? required class only used on labels, not on form controls itself

Reply:
yes, we have required for other inputs (input, email, number) and textarea:
https://github.com/osalabs/osafw-asp.net-core/blob/master/osafw-app/App_Data/template/common/form/showform/input.html

In other way the browser will not prevent empty values submission.

Reply:
ok, I remembered it - it's required attribute, not a class